include/deal.II/lac/dynamic_sparsity_pattern.h

```diff
@@ -420,6 +420,14 @@ class DynamicSparsityPattern : public Subscriptor
    */
   void symmetrize ();
 
+  /**
+   * Construct and store in this object the sparsity pattern corresponding to
+   * the product of @p right and @p left sparsity pattern.
```

Comment:
sorry for nitpicking, but `left` should come first to avoid confusion, because `result = left * right != right * left`.